tests/strictEqual.test.js

```diff
@@ -0,0 +1,82 @@
+const describe = require('mocha').describe
+const it  = require('mocha').it
+const expect = require('chai').expect
+const strictEqual = require('../strictEqual')
+
+describe('strictEqual', () => { 
+    describe('strictEqual', () => { 
+        it('returns true if values provided are same and are of the same type',() => {
+            // create test data
+
+            const valueOne = '5' 
+            const valueTwo = '5'
+
+
+
+            // call the function with the test data
+
+            const Return = strictEqual(valueOne,valueTwo)
+
+            //make assertions about what should be true
+
+            expect(Return).to.equal(true) 
+        })
+        it('returns false if values provided are different and are of  the same type',() => {
+            // create test data
+
+            const valueOne = 'a'
+            const valueTwo = '5' 
+
+
+
+            // call the function with the test data
```

Comment:
Comments are great to help us think through problems but always delete them when you are done so they don't pollute the repo history